src/core/algorithms.js

```diff
@@ -1,5 +1,5 @@
 /**
-Currently used only for adding 'assert' class to algorithm lists  
+Currently used only for adding 'assert' class to algorithm lists
```

Comment:
Maybe we should add `no-trailing-space` eslint rule.

Reply:
Yeah, would be good.